### PR TITLE
Add center plot window on the given label

### PR DESCRIPTION
**Pull request by JoshChristie**
Enable julia to interface with the center plot on dataset via label parameter. See displaz PR [#119](https://github.com/c42f/displaz/pull/119).

### CODE REVIEW

File: src/Displaz.jl [hunk 1]
```diff
@@ -314,6 +314,18 @@ function clearplot(plotobj::DisplazWindow, patterns...)
...
 clearplot(patterns...) = clearplot(current(), patterns...)
 
+"""
+    centerplot([plotobj::DisplazWindow=current()], label)
+
+Center current window on the first dataset given by the `label`. If not specified,
+`plotobj` is the current plot window.
+"""
+function centerplot(plotobj::DisplazWindow, label::AbstractString)
+    run(`displaz -script -centeronlabel $label`)
```
Comment (c42f) [inline src/Displaz.jl]: Should use `plotobj` to set the correct displaz window - see `plot3d`.

**Comment by c42f** [comment]
Great, thanks! There's one issue to be fixed, other than that this can be merged pending finalizing the `-centeronlabel` command.

**Comment by JoshChristie** [comment]
Thanks, issue should be fixed.

**Comment by c42f** [comment]
Hmm, was just about to merge this, but it struck me that we can choose a more generic name for the camera automation function so we can fold in the ability to give a rotation matrix, general affine transformation or label, depending on the type you give it.  (No need to implement it all here, just figure out a name.)  I'll have a think about whether there's a good name for this.

**Comment by JoshChristie** [comment]
Just noticed your commit/branch. I like your idea of `viewplot` :+1:

**Comment by c42f** [comment]
Ah yes, I was just writing a message.  So my attempt at generalizing in e294c8f seems ok?

Choosing a name turned out to be really hard, `viewplot()` was the best I could come up with.  `lookat` sort of makes sense, but not for the rotation part.

**Comment by JoshChristie** [comment]
Looks good to me. Having `view` in the name makes sense and I think `viewplot()` is short and sweet.
Other ideas:
- `cameraview()`\- camera might be the wrong word
- `positionplot()`\- may be confused with point positions

**Comment by c42f** [comment]
Thanks, I merged this as `viewplot()`.

**Pull request merged**